doc/source/user_guide/timeseries.rst

```diff
@@ -1780,6 +1777,50 @@ natural and functions similarly to :py:func:`itertools.groupby`:
 
 See :ref:`groupby.iterating-label` or :class:`Resampler.__iter__` for more.
 
+.. _timeseries.adjust-the-start-of-the-bins:
+
+Use `origin` or `offset` to adjust the start of the bins
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.1.0
+
+The bins of the grouping are adjusted based on the beginning of the day of the time series starting point. This works well with frequencies that are multiples of a day (like `30D`) or that divide a day evenly (like `90s` or `1min`). This can create inconsistencies with some frequencies that do not meet this criteria. To change this behavior you can specify a fixed Timestamp with the argument ``origin``.
+
+For example:
+
+.. ipython:: python
+
+    start, end = "2000-10-01 23:30:00", "2000-10-02 00:30:00"
+    middle = "2000-10-02 00:00:00"
+    rng = pd.date_range(start, end, freq="7min")
+    ts = pd.Series(np.arange(len(rng)) * 3, index=rng)
+    ts
+
+Here we can see that, when not using ``origin``, the result after "2000-10-02 00:00:00" are not identical depending on the start of time series:
+
+.. ipython:: python
+
+    ts.resample("17min").sum()
+    ts[middle:end].resample("17min").sum()
+
+
+Here we can see that, when using ``origin``, the result after "2000-10-02 00:00:00" are identical depending on the start of time series:
+
+.. ipython:: python
+
+   origin = pd.Timestamp("1970-01-01")
+   ts.resample("17min", origin=origin).sum()
+   ts[middle:end].resample("17min", origin=origin).sum()
+
+
+If needed you can just adjust the bins with an ``offset`` Timedelta that would be added to the default ``origin``.
+Those two examples are equivalent for this time series:
+
+.. ipython:: python
+
+    ts.resample("17min", origin=start).sum()
+    ts.resample("17min", offset=pd.Timedelta("23h30min")).sum()
```

Comment:
Hmm any chance you gave thought to requiring a negative Timedelta here? I'm on the fence

Reply:
Yes, negative Timedelta works actually, but not as someone may think with a frequency of 17min (since its a prime number) on this example. Let me explain with some example:

```python
import pandas._testing as tm

start = pd.Timestamp("2000-10-01 23:30:00")
ts_out_1  = ts.resample("17min", origin=start).sum()
ts_out_1b = ts.resample("17min", offset="23h30m").sum()
tm.assert_series_equal(ts_out_1, ts_out_1b)
ts_out_1
```
```
2000-10-01 23:30:00     9
2000-10-01 23:47:00    21
2000-10-02 00:04:00    54
2000-10-02 00:21:00    24
Freq: 17T, dtype: int64
```

So, as expected, `ts_out_1` and `ts_out_1b` is identical. The cause is simple: the default origin is the start of the day at midnight of "2000-10-01" on this example (since it is the beginning of the timeseries)

Applying a negative delta here would only be identical if you do this:
```python
default_origin = start.normalize()  #  '2000-10-01 00:00:00'
ts_out_2  = ts.resample("17min", origin=default_origin - pd.Timedelta("30m")).sum()
ts_out_2b = ts.resample("17min", offset="-30m").sum()
tm.assert_series_equal(ts_out_2, ts_out_2b)
ts_out_2
```
```
2000-10-01 23:18:00     0
2000-10-01 23:35:00    18
2000-10-01 23:52:00    27
2000-10-02 00:09:00    39
2000-10-02 00:26:00    24
Freq: 17T, dtype: int64
```

But `ts_out_2` and `ts_out_1` are not equal:

```python
tm.assert_series_equal(ts_out_1, ts_out_2)
```
```
AssertionError: Series are different

Series length are different
[left]:  4, DatetimeIndex(['2000-10-01 23:30:00', '2000-10-01 23:47:00',
               '2000-10-02 00:04:00', '2000-10-02 00:21:00'],
              dtype='datetime64[ns]', freq='17T')
[right]: 5, DatetimeIndex(['2000-10-01 23:18:00', '2000-10-01 23:35:00',
               '2000-10-01 23:52:00', '2000-10-02 00:09:00',
               '2000-10-02 00:26:00'],
              dtype='datetime64[ns]', freq='17T')
```


BUT this is only the case when dealing with weird frequencies! If you take a more classical frequency (that divide or is a multiple of a day) this is true:

```python
ts_out_3 = ts.resample("15min", offset="23h30m").sum()
ts_out_3b = ts.resample("15min", offset="-30m").sum()
tm.assert_series_equal(ts_out_3, ts_out_3b)
ts_out_3
```
```
2000-10-01 23:30:00     9
2000-10-01 23:45:00    21
2000-10-02 00:00:00    33
2000-10-02 00:15:00    45
Freq: 15T, dtype: int64
```